packages/react-wrapper/package.json

```diff
@@ -46,7 +46,7 @@
     "@types/react": "~16.9.0",
     "@types/react-dom": "~16.9.0",
     "commitizen": "~4.1.2",
-    "cypress": "~7.7.0",
+    "cypress": "~8.4.1",
```

Comment:
Revert the changes of this file please, we only need the ones in the lock files.!